woocommerce/woocommerce-gutenberg-products-block
# Cleanup withComponentId from base components

`componentId` is only necessary in `<OrderSelect />`, so it makes sense wrapping that component with `withComponentId` instead of wrapping the reviews blocks and passing `componentId` down as props.

### How to test the changes in this Pull Request:

1. Create a post with some reviews blocks and verify there are no JS errors when interacting with them in the editor.
2. Open the post in the frontend and verify you can still sort and _Load more_.
